DM-3545 Changelog entry / documentation for delete & bulk delete added (#2392)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-10-20-538-0-LWM2M-REST-documentation-additions.md

---
date:

> title: Documentation for REST-based LWM2M device registration

Reviewer: ```suggestion
title: Added documentation for REST-based LWM2M device registration
```

> title: Documentation for REST-based LWMM2M device registration

Reviewer 2: ```suggestion
title: Documentation for REST-based LWM2M device registration
```

change_type:

>   - value: change-VSkj2iV9m
>     label: Fix

Reviewer 2: ```suggestion
  - value: change-2c7RdTdXo4
    label: Improvement
```

product_area: Device Management & Connectivity
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-3545
version: 10.20.538.0
---

> Added documentation for registering and deleting LWM2M devices using the REST API.

Reviewer 2: ```suggestion
Added documentation for registering and deleting LWM2M devices using the LWM2M Service's REST API.
```

Author: Fixed, but differently. Never use 's on inanimate objects :)

https://style.mla.org/inanimate-nouns-and-possession/

> Added documentation for registering and deleting LWM2M devices using the REST API.

Reviewer: ```suggestion
In the user documentation, information has been added on how to register and delete LWM2M devices using the REST API, see [Registering LWM2M devices](/protocol-integration/lwm2m/#registering-lwm2m-devices).
```
